configs/opencv-ovms/scripts/run_instance_segmentation.sh

```diff
@@ -12,4 +12,12 @@ then
 	rmDocker=""
 fi
 
-docker run --network host --env-file <(env) --privileged $rmDocker -e DISPLAY=$DISPLAY -v ~/.Xauthority:/home/dlstreamer/.Xauthority -v /tmp/.X11-unix --name segmentation"$cid_count" instance-segmentation:dev python3 instance_segmentation_demo.py -m localhost:9000/models/instance_segmentation_omz_1040 --label coco_80cl_bkgr.txt -i $inputsrc --adapter ovms -t 0.85
\ No newline at end of file
+docker run --network host --env-file <(env) --privileged \
+$rmDocker -e DISPLAY=$DISPLAY -v ~/.Xauthority:/home/dlstreamer/.Xauthority \
+-v /tmp/.X11-unix --name segmentation"$cid_count" \
+-v $RUN_PATH/results:/tmp/results \
+instance-segmentation:dev \
+python3 instance_segmentation_demo.py -m localhost:9000/models/instance_segmentation_omz_1040 \
```

Comment:
```suggestion
python3 instance_segmentation_demo.py -m localhost:"$GRPC_PORT"/models/instance_segmentation_omz_1040 \
```
is 9000 GRPC_PORT? if so we better use the env instead of hard coded